### ISSUE TITLE
Boxplot in model interpretation

### ISSUE THREAD

**Issue by GaelVaroquaux** [INRIA/scikit-learn-mooc#239]
I would prefer a box plot instead of a swarm plot to show the variability of model coefficients in interpretation
Cc @glemaitre

**Comment by glemaitre** [comment]
Agreed. It seems that the line of the boxplot has been commented (no idea why). This example is directly taken from the scikit-learn page so we should update the code to avoid the warninng from seaborn as well (use a stipplot instand of the swarmplot)

```
sns.swarmplot(data=coefs, orient='h', color='k', alpha=0.5)
# sns.boxplot(data=coefs, orient='h', color='cyan', saturation=0.5)
```
